components/event-publisher-proxy/pkg/handler/handler_test.go

```diff
@@ -1,33 +1,32 @@
-//nolint:lll // this test uses many long lines directly from prometheus output
+//nolint:lll // output directly from prometheus
 package handler
```

Comment:
Replaced this file by: https://github.com/kyma-project/kyma/blob/main/components/event-publisher-proxy/pkg/handler/handler_v1alpha2_test.go